buildSrc/scripts/dackka_diff.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh +v
+
+errorOnInvalidDirectory() {
+  local directory=$1
+  local errorIfNotFound="${2:-Missing directory: $directory}"
+
+  if [ ! -d "$directory" ]; then
+    echo "[ERROR]: $errorIfNotFound" >&2
+    exit 1
+  fi
+}
+
+ROOT_DIRECTORY=../..
```

Comment:
would it make sense to check that this script is run directly from the `firebase-android-sdk` directory instead of from inside the scripts dir? IMO it would make sense since it looks for the `/build/` directory inside `firebase-android-sdk`